examples/automm/object_detection/coco_inference.py

```diff
@@ -0,0 +1,28 @@
+import time
+
+from autogluon.multimodal import MultiModalPredictor
+from autogluon.multimodal.utils import from_coco
+
+
+def test_coco_inference(checkpoint_name="faster_rcnn_r50_fpn_2x_coco"):
```

Comment:
Besides the data loading, `test_coco_inference` and `test_voc_inference` look the same. Consider combining these two with an additional argument like `dataset`.

Reply:
Done.